boards/xtensa/esp32s2_saola/board.cmake

```diff
@@ -1,3 +1,17 @@
 # SPDX-License-Identifier: Apache-2.0
 
+board_runner_args(openocd  --cmd-pre-init "set ESP_RTOS none")
+board_runner_args(openocd  --cmd-pre-init "set ESP32_ONLYCPU 1")
+
+# select FT2232 as default interface (as in ESP-WROVER-KIT or ESP-Prog)
+board_runner_args(openocd  --config "interface/ftdi/esp32s2_kaluga_v1.cfg")
+board_runner_args(openocd  --config "target/esp32s2.cfg")
+
+set (ESPRESSIF_OPENOCD_TOOL $ENV{ESPRESSIF_TOOLCHAIN_PATH}/openocd-esp32/bin/openocd)
```

Comment:
you add the espressif toolchain path to the CMakeCache here:
https://github.com/zephyrproject-rtos/zephyr/blob/0324997d1e7277374f7fcc734455eff55b7eeea1/cmake/toolchain/espressif/generic.cmake#L4

so why use the environment setting here ?

Reply:
Not necessary. Fixed.